### PR TITLE
#164798157 Set Up a Testing Environment

### PR DESCRIPTION
**Pull request by dorothyas**
**What does this PR do?**

- Sets up the testing environment for the app

**Description of Task to be completed?**

- Installing all necessary dependencies
- Writing tests for existing code

**How should this be manually tested?**

- Fetch this branch `git fetch origin ch-setup-testing-environment-164798157`
- Checkout to the branch `git checkout ch-setup-testing-environment-164798157`
- Run `npm install`
- Run `npm test` to run the available tests.
- Run `npm run test:coverage` for the coverage.

 **What are the relevant pivotal tracker stories?**

[#164798157](https://www.pivotaltracker.com/n/projects/2320796/stories/164798157)

**Screenshots**
Screenshot 2019-04-30 at 11 37 53

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "start": "webpack-dev-server --mode development --open --hot",
     "build": "webpack --mode production",
-    "lint": "eslint src --fix"
+    "lint": "eslint src --fix",
+    "test": "jest",
+    "test:watch": "jest --watch"
```
Comment (adesege) [inline package.json]: Add a script to collect coverage as well

**Review by adesege — CHANGES_REQUESTED** [review]
Good job. Please review

File: package.json (hunk 1 above)
Comment (dorothyas) [reply package.json]: I have worked on this feedback.

**Review by ezrogha — APPROVED** [review]
LGTM

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 .idea/
-node_modules/
\ No newline at end of file
+node_modules/
+coverage/
```
Comment (dannylwe) [inline .gitignore]: Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding "files.insertFinalNewline": true to your settings.json file

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (dorothyas) [reply .gitignore]: done

**Review by dannylwe — APPROVED** [review]
LGTM

**Review by adesege — APPROVED** [review]

**Pull request merged**